Doc/distributing/index.rst

```diff
@@ -119,47 +119,47 @@ recommended tools`_.
 
 .. _publishing-python-packages:
 
+
 Reading the Python Packaging User Guide
 =======================================
 
 The Python Packaging User Guide covers the various key steps and elements
 involved in creating and publishing a project:
 
-* `Project structure`_
-* `Building and packaging the project`_
-* `Uploading the project to the Python Packaging Index`_
+* `Creating the project structure`_
+* `Packaging the project`_
+* `Uploading the project to the Python Package Index`_
 * `The .pypirc file`_
 
-.. _Project structure: \
-    https://packaging.python.org/tutorials/distributing-packages/
-.. _Building and packaging the project: \
-   https://packaging.python.org/tutorials/distributing-packages/#packaging-your-project
-.. _Uploading the project to the Python Packaging Index: \
-   https://packaging.python.org/tutorials/distributing-packages/#uploading-your-project-to-pypi
+.. _Creating the project structure: \
+   https://packaging.python.org/tutorials/packaging-projects/#creating-the-package-files
+.. _Packaging the project: \
+   https://packaging.python.org/tutorials/packaging-projects/#generating-distribution-archives
+.. _Uploading the project to the Python Package Index: \
+   https://packaging.python.org/tutorials/packaging-projects/#uploading-the-distribution-archives
 .. _The .pypirc file: \
    https://packaging.python.org/specifications/pypirc/
 
 
-How do I...?
-============
+How do I…
```

Comment:
I rarely see English speakers use this character; I use it in personal writing but not in the Python docs.